src/Pim/Bundle/CatalogBundle/Elasticsearch/ProductIndexer.php

```diff
@@ -64,6 +66,33 @@ public function indexAll(array $products, array $options = [])
         }
 
         $this->indexer->bulkIndexes($this->indexType, $normalizedProducts, 'id');
+
+        if (!isset($options['associated_products']) || false === $options['associated_products']) {
```

Comment:
we need another way to do imo
this is not acceptable as is, I can't really explain why, but I feel danger :D 

we'll talk about it IRL this afternoon